src/components/Links/Links.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@edx/paragon';
+
+import './Links.scss';
+
+function Links(props) {
+  const { title, links } = props;
+
+  function handleLinkClick(e) {
```

Comment:
At least for now, I don't think we have a need for this click event handler as these are just external links. We might need to come back to this once we know more about our analytics need for this page. But, for now, we can probably remove this click event handler.